src/components/SkyBrowser/PaneSkybrowser.jsx

```diff
@@ -0,0 +1,83 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import { Resizable } from 're-resizable';
+import { size } from 'lodash';
+import MaterialIcon from '../common/MaterialIcon/MaterialIcon';
+import styles from './PaneSkybrowser.scss';
+import Pane from '../Sidebar/Pane';
+import Button from '../common/Input/Button/Button';
+import { excludeKeys } from '../../utils/helpers';
+
+class PaneSkybrowser extends Component {
+
+  render() {
+    const { children, title, closeCallback, setAsAttached, setAsDetached } = this.props;
+
+    return (
+      <section className={styles.pane}>
+        <Resizable
+        enable={{
```

Comment:
indendation: should be moved in 2 spaces fir all properties of `Resizable`

Also, this object (`{ top: false, ....}` an so on) is very big. maybe create a variable for it instead to make the returned object easier to read?